keyboards/handwired/chiron/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+SLEEP_LED_ENABLE = yes
```

Comment:
Could you please enable all these options in `keyboards/handwired/chiron/rules.mk`, and either empty or delete this file? This will enable QMK Configurator to compile firmware for this board, and have the firmware be identical whether it's done in Configurator or from source.

Reply:
Done.